### ISSUE TITLE
Ability to build the plugin using java 11

### ISSUE THREAD

**Issue by odockal** [redhat-developer/intellij-rsp#62]
[kind/Feature]

## Which functionality do you think we should add?
Ability to build the plugin using newer Java (11) in build.gradle script
```
compileJava {
    sourceCompatibility = '1.8'
    targetCompatibility = '1.8'
}
```

## Why is this needed?
Because older versions are outdated and more and more people uses newer platforms.

**Comment by odockal** [comment]
Seems to works when using proper ideaVersion...

**Comment by jeffmaury** [comment]
Fixed by 91669f686d02e0426d173dc0a933e6e08ca14d52 and fe4a29eb22fcb4a30b48a9095651737721c8f752
